queries/nginx/highlights.scm

```diff
@@ -0,0 +1,65 @@
+(comment) @comment @spell
+
+(value) @variable
+
+(attribute
+  (keyword) @attribute)
+
+[
+  (location_modifier)
+  "="
+] @operator
+
+[
+  (keyword)
+  "location"
+] @keyword
+
+[
+  "if"
+  "map"
+] @keyword.conditional
+
+(boolean) @boolean
+
+[
+  (auto)
+  (constant)
+  (level)
+  (connection_method)
+  (var)
+  condition: (condition)
+] @variable.builtin
+
+[
+  (file)
+  (mask)
+] @string.special.path
+
+[
+  (string_literal)
+  (quoted_string_literal)
+] @string
+
+(directive
+  (variable) @variable.parameter)
+
+(directive
+  (variable
+    (keyword) @variable.parameter))
```

Comment:
One of these should be enough. No need to capture both of them with the capture.